### ISSUE TITLE
Accessibility tab missingin DevTools

### ISSUE THREAD

**Issue by chriskirknielsen** [zen-browser/desktop#2046]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

The Accessibility tab in DevTools is missing. It is a critical tool in any developer's toolkit. It is available in Firefox.

**Firefox 131.0.2:**
<img width="484" alt="Firefox DevTools overflow menu showing Accessibility" src="https://github.com/user-attachments/assets/e70f828e-2009-41d2-97e4-991bb6c6bb3a">

**Zen 1.0.1-a.8:**
<img width="484" alt="Zen DevTools overflow menu not showing Accessibility" src="https://github.com/user-attachments/assets/7487211a-bb20-4d92-804a-14b832704e50">

Additionally, the "Show Accessible Properties" on right-click for a DOM element still appears but doesn't do anything, since it would normally take you to the Accessibility tab:
<img width="484" alt="image" src="https://github.com/user-attachments/assets/fdd9b242-2b7c-4988-9355-749e7c60a319">

Let me know if I need to provide more details.
Thanks!

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.8

### What platform are you seeing the problem on?

macOS - Intel

**Comment by pinage404** [comment]
The accessibility tab can be enabled in the devtools' settings

**Comment by chriskirknielsen** [comment]
> The accessibility tab can be enabled in the devtools' settings

Ugh, do I feel stupid… I thought this was enabled by default, didn't remember changing this in Firefox… until you pointed it out. Thank you!
